test/index.js

```diff
@@ -194,4 +195,39 @@ test('defaultComputeFileName', (t) => {
 	})
 })
 
+pTest('formatOptions are passed to sharp', co.wrap(function* (t) {
+	const file = yield vFile.read(src)
+
+	const file1 = file.clone()
+	const scale1 = {
+		format: 'webp',
+		maxWidth: 100,
+		formatOptions: {
+			quality: 80
+		},
+		maxWidth: Number.MAX_SAFE_INTEGER
+	}
+	const defaultFileSize = yield new Promise((res) => resize(file1, scale1, (err, newFile1) => {
+		t.ifError(err)
+		res(newFile1[SHARP_INFO].size)
+	}))
+
+	const file2 = file.clone()
+	const scale2 = {
+		format: 'webp',
+		maxWidth: 100,
+		formatOptions: {
+			quality: 100
+		},
+		maxWidth: Number.MAX_SAFE_INTEGER
+	}
+
+	resize(file2, scale2, (err, newFile2) => {
+		t.ifError(err)
+		t.ok(newFile2[SHARP_INFO].size > defaultFileSize)
+	});
+
+	t.end()
```

Comment:
Please move this to the below the `.size > defailtFileSize` check inside the callback.

Reply:
Whooops, that was silly, thanks

Reply:
ping (;

Reply:
I've put there yield so it will wait until resolved. I thought it would be better for reading.